### PR TITLE
lint: send-NFTs-across-chains

### LINKED ISSUE #1641: Lint Send NFTs Across Chains: https://wiki.iota.org/isc/how-tos/send-NFTs-across-chains/
**Is your feature request related to a problem? Please describe.**
There are some broken links, as well as references to ONFT V1 which is currently not being maintained 

**Describe the solution you'd like**
Do a lint pass over the How-To, fixing some minor issues and migrate anything related to V1

**Additional context**
This is related to: #1625 and #1634

### PR DESCRIPTION
**Pull request by salaheldinsoliman**
# Description of change

A linting pass over [cross chain NFT guide](docs/build/isc/v1.1/docs/how-tos/send-NFTs-across-chains.md)

## Links to any relevant issues

fixes #1641 

## Type of change
- Bug fix (a non-breaking change which fixes an issue)

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/build/isc/v1.1/docs/how-tos/send-NFTs-across-chains.md [hunk 1]
```diff
@@ -131,26 +131,20 @@ approve step is also required, but the operations will happen on the `ONFT` cont
 
 ##### LayerZero
 
-- [LayerZero Endpoint V1](https://docs.layerzero.network/v1/developers/technical-reference/mainnet/mainnet-addresses)
+- [LayerZero Endpoint V1 (Testnet)](https://docs.layerzero.network/v1/developers/evm/technical-reference/testnet/testnet-addresses)
+- [layerZero Endpoint V1 (Mainnet)](https://docs.layerzero.network/v1/developers/evm/technical-reference/mainnet/mainnet-addresses)
 - [LayerZero explorer](https://Testnet.layerzeroscan.com/)
 
-### Install the Library
+### Install and compile the Library
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/how-tos/send-NFTs-across-chains.md:138]:
```suggestion
### Install and Compile the Library
```

File: docs/build/isc/v1.3/docs/how-tos/send-NFTs-across-chains.md [hunk 2]
```diff
@@ -131,26 +131,20 @@ approve step is also required, but the operations will happen on the `ONFT` cont
 
 ##### LayerZero
 
-- [LayerZero Endpoint V1](https://docs.layerzero.network/v1/developers/technical-reference/mainnet/mainnet-addresses)
+- [LayerZero Endpoint V1 (Testnet)](https://docs.layerzero.network/v1/developers/evm/technical-reference/testnet/testnet-addresses)
+- [layerZero Endpoint V1 (Mainnet)](https://docs.layerzero.network/v1/developers/evm/technical-reference/mainnet/mainnet-addresses)
 - [LayerZero explorer](https://Testnet.layerzeroscan.com/)
 
-### Install the Library
+### Install and compile the Library
```
Comment (lucas-tortora) [inline docs/build/isc/v1.3/docs/how-tos/send-NFTs-across-chains.md:138]:
```suggestion
### Install and Compile the Library
```

**Review by lucas-tortora — CHANGES_REQUESTED** [review]

File: docs/build/isc/v1.1/docs/how-tos/send-NFTs-across-chains.md (hunk 1 above)
Comment (salaheldinsoliman) [reply docs/build/isc/v1.1/docs/how-tos/send-NFTs-across-chains.md:138]: Done

File: docs/build/isc/v1.3/docs/how-tos/send-NFTs-across-chains.md (hunk 2 above)
Comment (salaheldinsoliman) [reply docs/build/isc/v1.3/docs/how-tos/send-NFTs-across-chains.md]: Done

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**